### PR TITLE
docs: add org README

### PR DESCRIPTION
**Pull request by crowecawcaw**
### What was the problem/requirement? (What/Why)
1. The aws-deadline org has a lot repos, and it's not obvious what each of them is from their name (e.g. `deadline-cloud`).
2. Some people wander into the Deadline GitHub and aren't quite sure what's here or how it relates to the other documentation we have.
3. We don't have great documentation on DCC support, or how DCC support relates to fleets. See [this issue](https://github.com/aws-deadline/deadline-cloud-for-keyshot/issues/107#issuecomment-2304837916) as an example.

### What was the solution? (How)
Add an org README that will appear here to give quick pointers on how to navigate Deadline's open source stuff: https://github.com/aws-deadline/

### What is the impact of this change?
Hopefully it helps orient people who visit our GitHub pages for the first time.

### How was this change tested?
https://github.com/crowecawcaw/deadline-cloud.github

### Was this change documented?
Yes

### Is this a breaking change?
No
----

*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# AWS Deadline Cloud GitHub Org
```
Comment (ddneilson) [inline README.md]: It looks like you actually want this at the location: `profile/README.md`

See: https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-public-organization-profile-readme

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+**If you want to install Deadline Cloud tools on your workstation**, you can run the Deadline Cloud submitter installer. The installer can be found by going to the AWS Console > AWS Deadline Cloud > Downloads > Deadline Cloud submitter installer > Download.
+
+**If you're looking for documentation for the AWS Deadline Cloud service**, see the [AWS Deadline Cloud documentation](https://docs.aws.amazon.com/deadline-cloud/).
+
+**If you encounter a bug with an integration or tool**, please open an issue in the relevant repository.
+
+**If you want to build your own integration**, you can look through [deadline-cloud-samples](https://github.com/aws-deadline/deadline-cloud-samples) for  example job bundles, conda recipes, and other tooling. You can also look through the pre-built integrations below for larger examples. 
+
+**If you're looking for more information Open Job Description which Deadline uses for defining jobs**, see the [Open Job Description specification](https://github.com/OpenJobDescription/openjd-specifications/) or other repositories in the [Open Job Description GitHub org](https://github.com/OpenJobDescription/).
```
Comment (epmog) [inline profile/README.md]:
```suggestion
**If you're looking for more information on defining jobs**, see the [Open Job Description specification](https://github.com/OpenJobDescription/openjd-specifications/) or other repositories in the [Open Job Description GitHub org](https://github.com/OpenJobDescription/).
```

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Quick start
+
+**If you want to install Deadline Cloud tools on your workstation**, you can run the Deadline Cloud submitter installer. The installer can be found by going to the AWS Console > AWS Deadline Cloud > Downloads > Deadline Cloud submitter installer > Download.
+
+**If you're looking for documentation for the AWS Deadline Cloud service**, see the [AWS Deadline Cloud documentation](https://docs.aws.amazon.com/deadline-cloud/).
+
+**If you encounter a bug with an integration or tool**, please open an issue in the relevant repository.
+
+**If you want to build your own integration**, you can look through [deadline-cloud-samples](https://github.com/aws-deadline/deadline-cloud-samples) for  example job bundles, conda recipes, and other tooling. You can also look through the pre-built integrations below for larger examples.
```
Comment (epmog) [inline profile/README.md]:
```suggestion
**If you want to build your own integration**, you can look through [deadline-cloud-samples](https://github.com/aws-deadline/deadline-cloud-samples) for example job bundles, conda recipes, and other tooling. You can also look through the pre-built integrations below for larger examples. 
```

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+**If you encounter a bug with an integration or tool**, please open an issue in the relevant repository.
+
+**If you want to build your own integration**, you can look through [deadline-cloud-samples](https://github.com/aws-deadline/deadline-cloud-samples) for  example job bundles, conda recipes, and other tooling. You can also look through the pre-built integrations below for larger examples. 
+
+**If you're looking for more information Open Job Description which Deadline uses for defining jobs**, see the [Open Job Description specification](https://github.com/OpenJobDescription/openjd-specifications/) or other repositories in the [Open Job Description GitHub org](https://github.com/OpenJobDescription/).
+
+**If you want to contribute to an integration or library**, contributions are gratefully accepted!  You can open feature request issue to request a feature or to propose early feedback on an idea before raising a pull requests. Pull requests are welcomed  for features that would be widely useful to the community and which align with the existing codebase.
```
Comment (epmog) [inline profile/README.md]: nit: was a little word salad-y. Here's my attempt
```suggestion
**If you want to contribute to an integration or library**, contributions are gratefully accepted! You can open an issue to request a feature or receive early feedback on an idea before raising a pull request. Pull requests are welcomed for features that would be widely useful to the community and which align with the existing code base.
```

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+**If you want to adapt an existing integration to your specific workload**, consider forking a repository and making your changes in the fork.
+
+
+## Integrations
+Many repositories in this GitHub org are integrations with digital content creation software. See documentation in the individual repos for more info about features, usage, and compatability.
+
+Deadline Cloud supports service-managed fleets (SMF) and customer-managed fleets (CMF). With SMF, 
+Deadline fully manages the workers running the jobs and includes some pre-configured software. With CMF, the workers run on any customer-managed compute platform such as AWS EC2 or on-premise servers, and all configruation is handled by the customer. Many integrations work with SMF out of the box, and all integrations can be set up on CMF.
```
Comment (epmog) [inline profile/README.md]:
```suggestion
Deadline Cloud fully manages the workers running the jobs and includes some pre-configured software. With CMF, the workers run on any customer-managed compute platform such as AWS EC2 or on-premise servers, and all configuration is handled by the customer. Many integrations work with SMF out of the box, and all integrations can be set up on CMF.
```

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+| [SideFX Houdini](https://www.sidefx.com/) | [deadline-cloud-for-houdini](https://github.com/aws-deadline/deadline-cloud-for-houdini) | ✅ | ✅ |
+| [Unreal Engine](https://www.unrealengine.com/) | [deadline-cloud-for-unreal-engine](https://github.com/aws-deadline/deadline-cloud-for-unreal-engine) | ✅ |  |
+
+## Other notable repositories
+
+| Repository | Description |
+| - | - |
+| [deadline-cloud](https://github.com/aws-deadline/deadline-cloud) | A Python CLI and library for interacting with Deadline Cloud, especially submitting jobs. | 
+| [deadline-cloud-samples](https://github.com/aws-deadline/deadline-cloud-samples) | Example code and tools for running workloads on Deadline Cloud. | 
+| [deadline-cloud-worker-agent](https://github.com/aws-deadline/deadline-cloud-worker-agent) | Software that runs on workers and communicates with the Dealine API for receiving and completing tasks. |
```
Comment (epmog) [inline profile/README.md]:
```suggestion
| [deadline-cloud-worker-agent](https://github.com/aws-deadline/deadline-cloud-worker-agent) | Software that runs on workers and communicates with the Deadline Cloud API for receiving and completing tasks. | 
```

**Review by epmog — CHANGES_REQUESTED** [review]
Looks great! Some small fixes but nothing crazy

**Review by epmog — APPROVED** [review]

**Review by ddneilson — APPROVED** [review]

**Pull request merged**